docs/examples/matplotlib_interact_example.rst

```diff
@@ -0,0 +1,102 @@
+==============
+Matplotlib + Ipywidgets Interact Example
+==============
```

Comment:
The heading lines should be the same length as the heading.

Reply:
```
/home/moorepants/src/thebe/docs/examples/matplotlib_interact_example.rst:1: WARNING: Title overline too short.

==============
Matplotlib + Ipywidgets Interact Example
==============
```

build the docs locally:

```
cd docs
make html
```

Reply:
Ah, thanks for that. I'll build it locally and fix these issues.